android/src/main/java/com/facebook/react/modules/email/EmailModule.java

```diff
@@ -39,26 +39,29 @@ public void open(final String title) {
         // First create an intent with only the package name of the first registered email app
         // and build a picked based on it
         Intent intentChooser = pm.getLaunchIntentForPackage(ri.activityInfo.packageName);
-        Intent openInChooser = Intent.createChooser(intentChooser, title);
 
-        // Then create a list of LabeledIntent for the rest of the registered email apps
-        List<LabeledIntent> intentList = new ArrayList<LabeledIntent>();
-        for (int i = 1; i < resInfo.size(); i++) {
-            // Extract the label and repackage it in a LabeledIntent
-            ri = resInfo.get(i);
-            String packageName = ri.activityInfo.packageName;
-            Intent intent = pm.getLaunchIntentForPackage(packageName);
+        if (intentChooser != null) {
```

Comment:
Would it be cleaner to just do a return here (`if (intentChooser == null) return`), so we have less nesting?